CHANGELOG.md

```diff
@@ -1,3 +1,237 @@
+## v0.6.0 - 2020-12-31
+
+We're happy to announce Lokomotive v0.6.0 (Flying Scotsman).
+
+This release includes several new features, many component updates, and a new platform - [Tinkerbell](https://tinkerbell.org/).
+
+### Changes in v0.6.0
+
+- Remove deprecated CoreOS mentions from AWS ([#1245](https://github.com/kinvolk/lokomotive/pull/1245)) and bare metal ([#1246](https://github.com/kinvolk/lokomotive/pull/1246)).
+
+#### Kubernetes updates
+
+- Update Kubernetes to v1.19.4 and AKS to v1.18.10 ([#1189](https://github.com/kinvolk/lokomotive/pull/1189)).
+
+#### Component updates
+
+- Update External DNS to v0.7.4 ([#1115](https://github.com/kinvolk/lokomotive/pull/1115)).
+- Update Metrics Server to v2.11.2 ([#1116](https://github.com/kinvolk/lokomotive/pull/1116)).
+- Update Cluster Autoscaler to version v1.1.0 ([#1137](https://github.com/kinvolk/lokomotive/pull/1137)).
+- Update Rook to v1.4.6 ([#1117](https://github.com/kinvolk/lokomotive/pull/1117)).
+- Update Velero to v1.5.2 ([#1131](https://github.com/kinvolk/lokomotive/pull/1131)).
+- Update OpenEBS to v2.2.0 ([#1095](https://github.com/kinvolk/lokomotive/pull/1095)).
+- Update Contour to v1.10.0 ([#1170](https://github.com/kinvolk/lokomotive/pull/1170)).
+- Update Linkerd to stable-2.9.0 ([#1123](https://github.com/kinvolk/lokomotive/pull/1123)).
+- Update Web UI to v0.1.3 ([#1237](https://github.com/kinvolk/lokomotive/pull/1237)).
+- Update Prometheus Operator to v0.43.2 ([#1162](https://github.com/kinvolk/lokomotive/pull/1162)).
+- Update Calico to v3.17.0 ([#1251](https://github.com/kinvolk/lokomotive/pull/1251)).
+- Update aws-ebs-csi-driver to v0.7.0 ([#1135](https://github.com/kinvolk/lokomotive/pull/1135)).
+
+#### Terraform provider updates
+
+- Update Terraform providers to their latest versions ([#1133](https://github.com/kinvolk/lokomotive/pull/1133)).
+
+#### New platforms
+
+- Add support for Tinkerbell platform ([#392](https://github.com/kinvolk/lokomotive/pull/392)).
+
+#### Bug fixes
+
+- Add new worker pools when TLS bootstrap is enabled without remaining stuck ([#1181](https://github.com/kinvolk/lokomotive/pull/1181)).
+- Contour: Apply node affinity and tolerations to cert generator job also ([#1161](https://github.com/kinvolk/lokomotive/pull/1161)).
+- Improve hardware reservations validation rules on Equinix Metal ([#1186](https://github.com/kinvolk/lokomotive/pull/1186)).
+- Equinix Metal: Ensure Calico CRDs get created before use ([#1177](https://github.com/kinvolk/lokomotive/pull/1177)). They were occasionally failing.
+- openebs-operator: Set webhook policy to Ignore ([#1172](https://github.com/kinvolk/lokomotive/pull/1172)). This works around a bug in Kubernetes, see this [comment](https://github.com/openebs/openebs/issues/3046#issuecomment-642558701) for details.
```

Comment:
Inconsistency

Either have it `OpenEBS Operator` or `openebs-operator` and similiar for all components.

other example: Line 41
`Contour: ` or `contour:`